app_manager/appmanager/src/main/java/app/editors/manager/storages/dropbox/dropbox/login/DropboxLoginHelper.kt

```diff
@@ -0,0 +1,130 @@
+package app.editors.manager.storages.dropbox.dropbox.login
+
+import android.accounts.Account
+import android.content.Context
+import android.util.Log
+import androidx.lifecycle.*
+import app.documents.core.account.AccountDao
+import app.documents.core.account.CloudAccount
+import app.documents.core.settings.NetworkSettings
+import app.editors.manager.BuildConfig
+import app.editors.manager.app.App
+import app.editors.manager.storages.dropbox.dropbox.api.DropboxService
+import app.editors.manager.storages.dropbox.managers.utils.DropboxUtils
+import com.dropbox.core.DbxRequestConfig
+import com.dropbox.core.InvalidAccessTokenException
+import com.dropbox.core.android.Auth
+import com.dropbox.core.v2.DbxClientV2
+import com.dropbox.core.v2.users.FullAccount
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.launch
+import kotlinx.coroutines.withContext
+import lib.toolkit.base.R
+import lib.toolkit.base.managers.utils.AccountData
+import lib.toolkit.base.managers.utils.AccountUtils
+import javax.inject.Inject
+
+class DropboxLoginHelper(private val context: Context) {
```

Comment:
May be can add @Inject constructor and remove class from  module. Context will be taken from dagger graph